app/main.py

```diff
@@ -2,4 +2,69 @@ class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(self, name: str, weight: int, coords : list = None) -> None:
+
+        if coords is None:
+            coords = [0, 0]
+
+        self.name = name
+        self.weight = weight
+        self.coords = coords
+
+    def go_forward(self, step : int = 1) -> None:
```

Comment:
```suggestion
    def go_forward(self, step: int = 1) -> None:
```

Refactor in all cases